src/main/java/seedu/address/storage/JsonAdaptedLocation.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.location.Location;
+
+/**
+ * Json object for converting Locations.
+ */
+public class JsonAdaptedLocation {
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Location's %s field is missing!";
+    protected final String name;
+    protected final double lat;
+    protected final double lon;
```

Comment:
`latitude` and `longitude` would have been better names.

Reply:
Alright. Will change.